pkg/driver/controller_test.go

```diff
@@ -1220,6 +1383,79 @@ func TestControllerExpandVolume(t *testing.T) {
 	}
 }
 
+func TestIsShareableVolume(t *testing.T) {
+	testCases := []struct {
+		name     string
+		testFunc func(t *testing.T)
+	}{
+		{
+			name: "Test csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER",
+			testFunc: func(t *testing.T) {
+				testVolCap := []*csi.VolumeCapability{
+					{
+						AccessType: &csi.VolumeCapability_Mount{
+							Mount: &csi.VolumeCapability_MountVolume{},
+						},
+						AccessMode: &csi.VolumeCapability_AccessMode{
+							Mode: csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
+						},
+					},
+				}
+
+				if res := isShareableVolume(testVolCap); res {
+					t.Fatalf("Volume capability: %s is not shareable", testVolCap[0].AccessMode.GetMode().String())
+				}
```

Comment:
I see this code is repeated in all the 3 cases, this can be moved into below for loop section

Reply:
Fixed.